### PR TITLE
Use verbatim field-table file

### PR DESCRIPTION
**Pull request by maddenp-noaa**
**Synopsis**

For now, require `fv3` driver users to specify, as the `fv3.field_table.base_file` value, a path to a valid field-table file. Future work may add parsing/editing/dumping for this custom format, extending the existing `FieldTableConfig` class.

**Type**

- [x] Documentation
- [x] Enhancement (adds a new functionality)

**Impact**

- [x] This is a breaking change (changes existing functionality)

**Checklist**

- [x] I have added myself and any co-authors to the PR's _Assignees_ list.
- [x] I have reviewed the documentation and have made any updates necessitated by this change.

### CODE REVIEW

File: docs/sections/user_guide/yaml/field_table.rst
Comment (maddenp-noaa) [inline docs/sections/user_guide/yaml/field_table.rst:1]: We can restore (and possibly update) this later when we add field-table update support. For now, the official docs are probably the best place for users to learn about field-table files, and the `fv3` UW YAML component page has a link to those.

File: src/uwtools/resources/jsonschema/fv3.jsonschema [hunk 1]
```diff
@@ -34,84 +34,14 @@
...
-              "required": [
-                "update_values"
-              ]
-            }
-          ],
           "properties": {
             "base_file": {
               "type": "string"
-            },
-            "update_values": {
```
Comment (maddenp-noaa) [inline src/uwtools/resources/jsonschema/fv3.jsonschema:53]: We can revive this later, too.

File: src/uwtools/tests/drivers/test_fv3.py [hunk 2]
```diff
@@ -161,7 +163,7 @@ def test_FV3_namelist_file(driverobj):
 
 
 @pytest.mark.parametrize("domain", ("global", "regional"))
-def test_FV3_provisioned_run_directory_global(domain, driverobj):
+def test_FV3_provisioned_run_directory(domain, driverobj):
```
Comment (maddenp-noaa) [inline src/uwtools/tests/drivers/test_fv3.py:166]: I forgot to commit this in the last PR.

File: src/uwtools/tests/test_schemas.py [hunk 3]
```diff
@@ -238,64 +218,6 @@ def test_schema_fv3_domain(fv3_prop):
...
-        with_set(val1, "a string", "foo", "profile_type", "surface_value")
-    )
-    # top_value is required if name is "profile":
-    assert "'top_value' is a required property" in errors(
-        with_del(val2, "bar", "profile_type", "top_value")
-    )
-    # top_value is numeric:
-    assert "'a string' is not of type 'number'" in errors(
-        with_set(val2, "a string", "bar", "profile_type", "top_value")
-    )
```
Comment (maddenp-noaa) [inline src/uwtools/tests/test_schemas.py:296]: These schema tests can be revived when the corresponding schema is revivied.

**Review by christinaholtNOAA — APPROVED** [review]
LGTM. Thanks for the quick turnaround on the demolition! ⚒️ 💥

**Pull request merged**